.travis.yml

```diff
@@ -1,13 +1,41 @@
-before_script:
-        - curl -O -L http://prdownloads.sourceforge.net/sbcl/sbcl-1.2.6-x86-64-linux-binary.tar.bz2
-        - tar xjf sbcl-1.2.6-x86-64-linux-binary.tar.bz2
-        - pushd sbcl-1.2.6-x86-64-linux/ && sudo bash install.sh && popd
-        - curl -O -L http://beta.quicklisp.org/quicklisp.lisp
-        - sbcl --load quicklisp.lisp --eval '(quicklisp-quickstart:install)' --eval '(quit)'
-        - curl -OL http://ccl.clozure.com/ftp/pub/release/1.10/ccl-1.10-linuxx86.tar.gz
-        - tar xzf ccl-1.10-linuxx86.tar.gz
-        - export PATH=`pwd`/ccl:$PATH
-#        - lx86cl64 -b --load quicklisp.lisp --eval '(progn (quicklisp-quickstart:install) (quit))'
+language: common-lisp
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - libc6-i386
+      - clisp
+      - openjdk-7-jre
```

Comment:
this doesn't exist.

https://travis-ci.org/pnathan/cl-ansi-text/jobs/624632302?utm_mediumnotification&utm_source=github_status

```
apt-get install failed
The command "sudo -E apt-get -yq --no-install-suggests --no-install-recommends $(travis_apt_get_options) install libc6-i386 clisp openjdk-7-jre" failed and exited with 100 during .
```

Reply:
not actually sure what the latest Java ABCL supports is, I know there's been work on it, but I haven't been dialed in lately.

Reply:
Ah I forgot to remove it. The package existed in the past but seems to have been removed from Travis at some point.

Reply:
It is not required either.